sys/include/net/ng_netreg.h

```diff
@@ -126,6 +127,26 @@ int ng_netreg_num(ng_nettype_t type, uint32_t demux_ctx);
  */
 ng_netreg_entry_t *ng_netreg_getnext(ng_netreg_entry_t *entry);
 
+/**
+ * @brief   Builds a header for sending for a protocol in the registry and
```

Comment:
"Builds a header for sending and adds it to the packet buffer" would also work, I think. That the protocol is defined through `type` is apparent through the type parameter I think.